src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-attributes-update.js

```diff
@@ -143,7 +143,12 @@ export default class EzBtnAttributesUpdate extends EzWidgetButton {
             return;
         }
 
-        block.$.classList.remove(...this.classes.choices);
+        const classList = block.$.classList;
+        this.classes.choices.forEach(className => {
```

Comment:
```suggestion

        this.classes.choices.forEach((className) => {
```

Reply:
@dew326 done in https://github.com/ezsystems/ezplatform-admin-ui/pull/1194/commits/0ecdbd06ba4abe66a99aa40670295d5befefa12d